sdk/monitor/Azure.Monitor.Query/src/Models/LogsQueryAudience.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using Azure.Core;
+
+namespace Azure.Monitor.Query
+{
+    /// <summary> Cloud audiences available for Query. </summary>
+    public readonly partial struct LogsQueryAudience : IEquatable<LogsQueryAudience>
+    {
+        private readonly string _value;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LogsQueryAudience"/> object.
+        /// </summary>
+        /// <param name="value">The Microsoft Entra audience to use when forming authorization scopes. For the language service, this value corresponds to a URL that identifies the Azure cloud where the resource is located. For more information: <see href="https://learn.microsoft.com/azure/azure-government/documentation-government-cognitiveservices" />.</param>
```

Comment:
@scottaddie I believe the cognitiveservices link is wrong - could I change this to the ReadMe?

Reply:
Yes, this can link to a specific section in the README. You'd want to point to the published version on learn.microsoft.com.